katas/main/src/pokerhands/jorge/HandScore.cs

```diff
@@ -0,0 +1,118 @@
+﻿namespace PokerGame.Hands.Jorge;
+using System.Linq;
+
+public class PokerHand
+{
+    public List<string> pairArray = new List<string>(); // List to know pairs.
+
+    public double HandToPlay(string listPokerHand)
```

Comment:
If possible, try to break this method in smaller pieces, so each piece can have its own unit test and describes what is the goal defined for it.